packages/vm/src/conversion.rs

```diff
@@ -63,7 +63,7 @@ mod test {
                 assert_eq!(to_type, "u32");
                 assert_eq!(input, "4294967296");
             }
-            Err(err) => panic!("unexpected error: {:}", err),
+            Err(err) => panic!("unexpected error: {:?}", err),
```

Comment:
I was more curious as to what `{:}` meant, still can't really decipher it reading https://doc.rust-lang.org/std/fmt/
Since these are snafu errors, they actually have a nice display string and `{}` will give good text. 
Since this is just s test caseI have no strong opinon between the two.